app/src/main/java/org/oppia/app/profileprogress/ProfileProgressActivityPresenter.kt

```diff
@@ -21,4 +34,25 @@ class ProfileProgressActivityPresenter @Inject constructor(private val activity:
   private fun getProfileProgressFragment(): ProfileProgressFragment? {
     return activity.supportFragmentManager.findFragmentById(R.id.profile_progress_fragment_placeholder) as ProfileProgressFragment?
   }
+
+  fun showPictureEditDialog() {
+    val previousFragment =
+      activity.supportFragmentManager.findFragmentByTag(TAG_PROFILE_PICTURE_EDIT_DIALOG)
+    if (previousFragment != null) {
+      activity.supportFragmentManager.beginTransaction().remove(previousFragment).commitNow()
+    }
+    val dialogFragment = ProfilePictureEditDialogFragment.newInstance()
+    dialogFragment.showNow(activity.supportFragmentManager, TAG_PROFILE_PICTURE_EDIT_DIALOG)
+  }
+
+  fun openGalleryIntent() {
+    val galleryIntent = Intent(Intent.ACTION_PICK, MediaStore.Images.Media.EXTERNAL_CONTENT_URI)
+    activity.startActivityForResult(galleryIntent, GALLERY_INTENT_RESULT_CODE)
+  }
+
+  fun handleOnActivityResult(intent: Intent?) {
+    intent?.let {
+      profileManagementController.updateProfileAvatar(profileId,intent.data, /* colorRgb= */-10710042)
```

Comment:
This is still not correct, you missed space between `profileId` and `intent.data` and also just after inline comment,  please copy below mentioned line and paste it.

`profileManagementController.updateProfileAvatar(profileId, intent.data, /* colorRgb= */ -10710042)`

Reply:
Okay got it will not happen